src/app/pages/user/user.component.ts

```diff
@@ -69,11 +69,11 @@ export class UserComponent implements OnInit, AfterViewInit {
       }
     });
   }
-  updateUser(_id: number, firstname: string, lastname: string, email: string, username: string) {
+  updateUser(_id: number, firstname: string, lastname: string, email: string, username: string, role: any, Class: any) {
     const dialogRef = this.dialog.open(UpdateUserDialogComponent, {
-      data: { _id, firstname, lastname, email, username },
-      height: '368px',
-      width: '630px',
+      data: { _id, firstname, lastname, email, username, role, Class },
```

Comment:
as above, can we pass object here instead? 
also, just out of curiosity, why `Class` is with capital `C`